templated_output/Arkansas_Nuclear_2.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation>
```

Comment:
These disappear during xml inclusion. I needed a base tag so that no xml fragments occur.
I can change them to any other name but the only things that get added during xml inclusion is the children of tag.